compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/IntermediateTextInputUIView.uikit.kt

```diff
@@ -471,39 +486,52 @@ internal class IntermediateTextInputUIView(
         }
     }
 
+    private fun shouldReloadContextMenuItems(actions: TextActions): Boolean {
+        return (_currentTextMenuActions?.copy == null) != (actions.copy == null) ||
+            (_currentTextMenuActions?.paste == null) != (actions.paste == null) ||
+            (_currentTextMenuActions?.cut == null) != (actions.cut == null) ||
+            (_currentTextMenuActions?.selectAll == null) != (actions.selectAll == null)
+    }
+
+    private fun cancelContextMenuUpdate() {
+        menuMonitoringJob.cancel()
+        menuMonitoringJob = Job()
+    }
+
     /**
      * Show copy/paste text menu
      * @param targetRect - rectangle of selected text area
      * @param textActions - available (not null) actions in text menu
      */
     fun showTextMenu(targetRect: org.jetbrains.skia.Rect, textActions: TextActions) {
-        _currentTextMenuActions = textActions
-        val menu: UIMenuController = UIMenuController.sharedMenuController()
         val cgRect = CGRectMake(
             x = targetRect.left.toDouble(),
             y = targetRect.top.toDouble(),
             width = targetRect.width.toDouble(),
             height = targetRect.height.toDouble()
         )
         val isTargetVisible = CGRectIntersectsRect(bounds, cgRect)
+
         if (isTargetVisible) {
-            if (menu.isMenuVisible()) {
-                menu.setTargetRect(cgRect, this)
```

Comment:
`showTextMenu` can be called multiple times, even if the context menu is already shown.

In the new code we will call `showMenuFromView` second time. Will it work?